stores/moderationSubmissionsStore.ts

```diff
@@ -101,6 +101,27 @@ export const useModerationSubmissionsStore = defineStore(
             }
         }
 
+        async function rejectSubmission() {
+            const facilityInputVariables = {
```

Comment:
similar to your comment on the other PR, let's add a check or log here for selectedSubmissionId